### DISCUSSION TITLE
How to install in a linux machine?

### DISCUSSION THREAD

**Discussion by louiekwan** [GoogleCloudPlatform/terraformer#926] [General]
Any installation documentation?

$ terraformer import aws --resources=vpc,subnet --connect=true --regions=ca-central-1
2021/05/11 12:54:16 aws importing region ca-central-1
2021/05/11 12:54:16 open C:\Users\abc\.terraform.d/plugins/windows_amd64: The system cannot find the path specified.

In a linux machine.

[ec2-user@ip-10-x-x-x terraformer]$ terraformer import aws --resources=vpc,subnet --connect=true --regions=ca-central-1
2021/05/11 16:55:07 aws importing region ca-central-1
2021/05/11 16:55:07 open /home/ec2-user/.terraform.d/plugins/linux_amd64: no such file or directory

**Comment by MattSnow-amd** [comment]
You need to download the aws provider plugin then copy it to /home/ec2-user/.terraform.d/plugins/linux_amd64 .

**Comment by sunil29feb** [comment]
Download aws terrafrom plugin by creating version.tf and running terraform init
1. content of version.tf -
terraform {
  required_providers {
    aws = {
      source  = "hashicorp/aws"
      version = "~> 3.0"
    }
  }
}
2. Run Terraform init
